state/validation_test.go

```diff
@@ -282,22 +284,26 @@ func TestValidateFailBlockOnCommittedEvidence(t *testing.T) {
 func TestValidateAlreadyPendingEvidence(t *testing.T) {
 	var height int64 = 1
 	state, stateDB, _ := makeState(2, int(height))
+	addr, _ := state.Validators.GetByIndex(0)
+	addr2, _ := state.Validators.GetByIndex(0)
+	ev := types.NewMockEvidence(height, time.Now(), addr)
+	ev2 := types.NewMockEvidence(height, time.Now(), addr2)
+
+	evpool := &mocks.EvidencePool{}
+	evpool.On("IsPending", mock.AnythingOfType("types.MockEvidence")).Return(func(e types.Evidence) bool {
+		return e.Equal(ev)
+	})
+	evpool.On("IsCommitted", mock.AnythingOfType("types.MockEvidence")).Return(false)
 
-	evpool := evmock.NewDefaultEvidencePool()
 	blockExec := sm.NewBlockExecutor(
 		stateDB, log.TestingLogger(),
 		nil,
 		nil,
 		evpool)
 	// A block with a couple pieces of evidence passes.
 	block := makeBlock(state, height)
-	addr, _ := state.Validators.GetByIndex(0)
-	addr2, _ := state.Validators.GetByIndex(0)
-	// add pending evidence
-	pendingEv := evpool.AddMockEvidence(height, addr)
```

Comment:
where did this go?

Reply:
Also replaced this with the testify/mock